### PR TITLE
gh-94485: Set line number of module's RESUME instruction to 0, as specified by PEP 626.

### LINKED ISSUE #94485: frame.f_lineno can be None
**Bug report**

As far as I know `frame.f_lineno` should never be `None`, yet, in the case below it is:

```
import sys
def tracefunc(frame, event, arg):
    if frame.f_lineno is None:
        raise AssertionError('frame.f_lineno is None!', frame)
    return tracefunc
sys.settrace(tracefunc)

# Apparently anything we import would give a frame.f_lineno = None during tracing.
import threading
```

**Your environment**

- CPython versions tested on: `3.11.0b3 (main, Jun  1 2022, 13:29:14) [MSC v.1932 64 bit (AMD64)]`
- Operating system and architecture: `Windows`

### PR DESCRIPTION
**Pull request by iritkatriel**
Closes #94485.

* Issue: gh-94485

### CODE REVIEW

File: Misc/NEWS.d/next/Core and Builtins/2022-07-01-20-00-19.gh-issue-94485.mo5st7.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Line number of a module's ``RESUME`` instruction is set to 0 as specified in
+:pep:`626`.  A
```
Comment (arhadthedev) [inline Misc/NEWS.d/next/Core and Builtins/2022-07-01-20-00-19.gh-issue-94485.mo5st7.rst]: It looks like the last line is undrecopied.

Comment (iritkatriel) [inline Misc/NEWS.d/next/Core and Builtins/2022-07-01-20-00-19.gh-issue-94485.mo5st7.rst]:
```suggestion
:pep:`626`.
```

**Comment by pablogsal** [comment]
@iritkatriel I think you need to regenerate `frozenmain.c`

**Comment by iritkatriel** [comment]
It's too messy when I try to work on Mark's branch. I'll create a new PR.

**Pull request closed without merging**